### PR TITLE
Add detach volume

### PR DESCRIPTION
**Pull request by Diptipowervs**
This in response to: https://github.com/powervs-ibm/terraform-provider-ibm/issues/57

### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request

Relates OR Closes #0000

Output from acceptance testing:

```
$ make testacc TESTARGS='-run=TestAccXXX'

...
```

### CODE REVIEW

File: website/docs/r/pi_volume_attach.html.markdown [hunk 1]
```diff
@@ -8,7 +8,7 @@ description: |-
 ---
 
 # ibm_pi_volume_attach
-Attaches volume to a Power Systems Virtual Server instance. For more information, about managing volume, see [getting started with IBM Power Systems Virtual Servers](https://cloud.ibm.com/docs/power-iaas?topic=power-iaas-getting-started).
+Attach and Detach volume to a Power Systems Virtual Server instance. For more information, about managing volume, see [getting started with IBM Power Systems Virtual Servers](https://cloud.ibm.com/docs/power-iaas?topic=power-iaas-getting-started).
```
Comment (ismirlia) [inline website/docs/r/pi_volume_attach.html.markdown:11]:
```suggestion
Attach and Detach volumes to a Power Systems Virtual Server instance. For more information, about managing volume, see [getting started with IBM Power Systems Virtual Servers](https://cloud.ibm.com/docs/power-iaas?topic=power-iaas-getting-started).
```

File: website/docs/r/pi_volume_attach.html.markdown [hunk 2]
```diff
@@ -19,6 +19,11 @@ resource "ibm_pi_volume_attach" "testacc_volume_attach"{
 	pi_volume_id = "<id of the volume to attach>"
 	pi_instance_id = "<pvm instance id>"
 }
+resource "ibm_pi_volume_detach" "testacc_volume_detach"{
+	pi_cloud_instance_id = "<value of the cloud_instance_id>"
+	pi_volume_id = "<id of the volume to detach>"
+	pi_instance_id = "<pvm instance id>"
+}
```
Comment (ismirlia) [inline website/docs/r/pi_volume_attach.html.markdown:26]: There is no ibm_pi_volume_detach resource. You use the ibm_pi_volume_attach resource to do both.

**Comment by michaelkad** [comment]
After looking carefully at the code, I think we don't have to change this because it is tightly couple. We will have to have a new detach. I think we have to close this.

**Comment by Diptipowervs** [comment]
Close PR

**Comment by Diptipowervs** [comment]
Close PR

**Pull request closed without merging**